alpine/Dockerfile

```
@@ -0,0 +1,44 @@
+FROM debian:stretch-slim AS zip_downloader
+LABEL maintainer="NebulousLabs <developers@nebulous.tech>"
+
+ARG SIA_VERSION="1.4.7"
+ARG SIA_PACKAGE="Sia-v${SIA_VERSION}-linux-amd64"
+ARG SIA_ZIP="${SIA_PACKAGE}.zip"
+ARG SIA_RELEASE="https://sia.tech/releases/${SIA_ZIP}"
+
+RUN apt-get update && apt-get install -y wget unzip
+
+RUN wget "$SIA_RELEASE" && \
+      mkdir /sia && \
+      unzip -j "$SIA_ZIP" "${SIA_PACKAGE}/siac" -d /sia && \
+      unzip -j "$SIA_ZIP" "${SIA_PACKAGE}/siad" -d /sia
+
+FROM alpine:3
+LABEL maintainer="NebulousLabs <developers@nebulous.tech>"
```

```suggestion
LABEL maintainer="NebulousLabs <devs@nebulous.tech>"
LABEL autoheal=true
```
If you're adding the HEALTHCHECK you might as well label the container for autohealing. We also changed our email (there is an open PR).

---

Fixed in e209584.